packages/ListBoxBrowser/src/ListBoxBrowser.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { container, flex } from "./ListBoxBrowser.styles";
+import * as sc from "./ListBoxBrowser.styles";
```

Comment:
why `*` instead of the actual names?

Reply:
We ended up at this conclusion for readability + convenience. It's convenient to have one `import` that you don't have to mess with as you build out / refactor the component.  And it's more readable to identify styled components with a `sc.` prefix on them (and then you can export something without a `styled` prefix / suffix).  So in the `styles.js` you can `export Button`, and then in the JSX, you can use `<sc.Button>`.